schemas/string_interpolation/definitions.json

```diff
@@ -37,6 +37,15 @@
       }
     ]
   },
+  "summary": {
+    "description": "TextField string concatenations for the summary page",
+    "properties": {
+      "concatenation_type": {
+        "enum": ["NewLine", "Space"]
```

Comment:
Should be `Newline`?

Reply:
Down as NewLine on the story

Reply:
I think we should also change `NewLine` to `Newline` as a newline character is 1 word.